codes/quantum/qubits/monitored_random_circuits.yml

```diff
@@ -0,0 +1,94 @@
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: monitored_random_circuits
+physical: bits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: 'Monitored Random Circuits'
+
+# First reference with the code (optional).
+introduced: '\cite{10.1103/PhysRevX.9.031009}, \cite{10.1103/PhysRevB.98.205136}, \cite{arXiv:1808.06134}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A monitored random circuit is described by a series of intermittant local projective Pauli measurements with unitary time evolution operators. When the rate of projective measurements is independently controlled by a probability parameter p, there exist two sable phases, one described by the volume-law entanglement entropy and the other described by area-law entanglement entropy. The phase transition can be understood from the perspective of quantum error correction and information scrambling, as discussed in \cite{arXiv:1903.05124}. In the volume-law phase, the random unitaries scramble the information (rotate the code words), protecting the information from the projective measurements (aka errors). When the rate of measurements surpasses a threshold \( p_c \) the information is no longer protected. The key insight offered by the quantum error correction implemented by the scrambling dynamics is that the entanglement and correlation between subsystems A, B is scrambled into non-local degrees of freedom and cannot be disentangled by local measurements below \( p < p_c \).'
+
+- '\textbf{Clifford monitored random circuits}: When considering unitary evolutions, it can be useful to only condier unitaries that belong to the Clifford group such that the series of time evolved Pauli operators are always Pauli operators, by definition of the Clifford group. In this case, the time-evolved projective measurement at time step $j$ is given by \( P_j = U_{\tau} \cdots U_j P_j^’ (U_{\tau} \cdots U_j)^{\dagger} \). Thus the monitored random circuit just implements a series of projective measurements at various time steps.'
+
+protection: 'When in the volume-law phase, protects against random projective measurements at a rate \( p < p_c \). As described in \cite{1903.05124} and \cite{2105.13352} the subleading contribution to the entropy can be interpreted as the code distance and is given by \(d_{\rm code} = L^{\gamma_{code}} \) where \( \gamma_{code} \approx 0.38 \) and \( L \) is the size of the system. For a monitored circuit in the volume law-phase with an exponentially long memory time, \( d \leq Poly(n) \).'
```

Comment:
The reference to 1903.05124 is not correct.  These observations were first made in 1905.05195.  To be more precise, the sub-leading term does not capture the code-distance, but the contiguous code distance (see arxiv:0810.1983).  The actual code distance is not known, but is conjectured to be proportional to the contiguous code distance.